### DISCUSSION TITLE
[BUG] Add-PnPContentTypesFromContentTypeHub -ContentTypes missing column

### DISCUSSION THREAD

**Discussion by Wernbe** [pnp/powershell#3140] [General]
### Reporting an Issue or Missing Feature
In some cases when creating a team and when using Add-PnPContentTypesFromContentTypeHub -ContentTypes to add an contenttype to the sharepoint site->list. One of the columns of the contenttype is missing

### Expected behavior 
In most of the cases the contenttype from the hub has all the columns. 

### Actual behavior 
One of the columns is missing.

### Steps to reproduce behavior
Add-PnPContentTypesFromContentTypeHub -ContentTypes "0x01006E4EB9BA6BA49944B4D15C7BC50B646D"

### What is the version of the Cmdlet module you are running?
Running 2.1.1

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows
- [ ] Linux
- [X ] MacOS
- [ ] Azure Cloud Shell
- [ X] Azure Functions

**Comment by Wernbe** [comment]
I just found a quick fix. Add a sitecolumn to the site with the same name as missing. Then try to add the Add-PnPContentTypesFromContentTypeHub again. You then get an error stating that there is conflicten sitecolumn. Then you remove the sitecolumn again and then when using Add-PnPContentTypesFromContentTypeHub again the missing column is also available :-)

**Comment by gautamdsheth** [comment]
hi @Wernbe , looking at your workaround, the problem seems to be coming from the server side and not the cmdlet itself. Would recommend reaching out to Microsoft support for help, we can't do much here, so will move it to discussions.

You can ask them to look at the cmdlet code here:

https://github.com/pnp/powershell/blob/dev/src/Commands/ContentTypes/AddContentTypesFromContentTypeHub.cs

**Reply by Wernbe** [reply]
@gautamdsheth Thanks Gautam, We will if the problem arises again.
